### ISSUE TITLE
Can't run any of the sample in Android Device or Automotive emulator

### ISSUE THREAD

**Issue by Swati90** [android/car-samples#17]
![Screenshot_1624432090](https://user-images.githubusercontent.com/7079188/123051989-f5551280-d41f-11eb-9a8f-bbb079bf9b9a.png)
![Screenshot_1624432090](https://user-images.githubusercontent.com/7079188/123052069-09990f80-d420-11eb-9787-a4bfea13f928.png)

**Comment by mani-w3w** [comment]
Yup same problem here I reported in this issue #16, anyone can help us? They have updated last week to` implementation "androidx.car.app:app-automotive:1.1.0-alpha01" and "androidx.car.app:app:1.1.0-alpha01"` and I think the readme is not up to date here, or we need some new emulator that we don't have? not really sure how to run it now

**Comment by Greathyx** [comment]
Same here. I am using Android Studio 2.4.1, but I still got the problem.

**Comment by mani-w3w** [comment]
@kokoro-team @yjm607 can someone help us with this?

**Comment by ikbalkaya** [comment]
Same here

**Comment by SamuelBrucksch** [comment]
Try version 1.0.0 instead of 1.1.0-alpha01 or enable beta in playstore for android auto.

**Comment by derjohng** [comment]
Same here. And, I got the error message as below.
E/CarApp.Act: No handlers found for intent: Intent { act=android.car.template.host.RendererService }

**Comment by mani-w3w** [comment]
From what I understood (and Google added to the readme) they released this new way of building automotive apps using Screens/templates but then they don't have a public **Host** to render these templates available yet. 

Basically, the new automotive apps based on Screens/Templates (instead of activities/fragments) will not run on android automotive emulators for god knows how long? Find this really weird, how do we test the apps then?

**Comment by vy12021** [comment]
There hav a new error when a run the sampe app on the latest AAOS emulator (aaos for api level 31 reversion 2), as below is the error log: 
`CarApp.Act: Host service ComponentInfo{com.google.android.apps.automotive.templates.host/com.google.android.apps.automotive.templates.host.stub.ServiceStub} rejected the binding request`

and the error screenshot is still the top issue presented.

the error on the aaos for API level 31 reversion 1 as below:
`E/CarApp.Act: No handlers found for intent: Intent { act=android.car.template.host.RendererService }`

so, I guess that the render service is implemented on the latest AAOS build, but something is still wrong, 
really need the aaos team for help currently.

**Comment by vy12021** [comment]
The sample app works on the AAOS after updating the Host App to the latest via the Play store, You can try the README file section for details: [Run the Automotive Apps in the Automotive OS Emulator](https://github.com/android/car-samples/tree/main/car_app_library#run-the-automotive-apps-in-the-automotive-os-emulator), special the update guide.
